packages/js-client/test-environment.js

```diff
@@ -0,0 +1,16 @@
+const Environment = require('jest-environment-jsdom');
+
+module.exports = class CustomTestEnvironment extends Environment {
```

Comment:
Why is this needed? to run the IPFS client on NodeJS as if it was a browser?

Reply:
Yes, we added this at the begining so I dont remember exactly but I think we used some browser only sutuff from js and we needed to addd the shims fopr the tests to work on the nodejs env